### ISSUE TITLE
Add search support for phrase proximity

### ISSUE THREAD

**Issue by mlissner** [freelawproject/courtlistener#2968]
I don't know if this is possible, but in commercial systems (West, Lexis, etc), you can do a query like:

`"one two" ~10 "three four"`

And you'll get back results where the phrase "one two" is within 10 tokens of "three four". I don't think we support this, and I'm not sure there's a way do to so, but I want to open this issue to discuss. 

In our system, I think the query is interpreted as:

> ("one" within 10 tokens of "two") AND the phrase "three four".

**Comment by albertisfu** [comment]
It seems we can close this issue, as we concluded in https://github.com/freelawproject/courtlistener/issues/3888 that it's not possible to implement.

**Comment by mlissner** [comment]
Yep. Closing for now. Hopefully somebody can find a way someday.
